admin/templates/users/user.html

```diff
@@ -218,6 +218,19 @@ <h3>User details</h3>
                         {% endfor %}
                     </td>
                 </tr>
+                <tr>
+                <td>
+                    OSF Groups
+                </td>
+                <td>
+                    <select class="form-control" style="width:90%; display:inherit;" id="groups-select">
+                        {% for group in user.osf_groups %}
+                            <option value="{{ group | reverse_osf_group }}">{{group.name}} ({{group.id}})</option>
```

Comment:
I'd update the User serializer to return the group's role on the node, which I think would be useful info, and pass in group id instead of group object.
```
{% for group in user.groups %}
    <option value="{{ group.id | reverse_osf_group }}">{{group.name}} ({{group.role}})</option>
```